src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -128,6 +128,10 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
       caseEventData.ignore_warning = this.ignoreWarning;
       caseEventData.event_data = this.editForm.value.data;
 
+      if (this.caseEdit.caseDetails) {
```

Comment:
Can this be covered by an unit test? Any expect in existing test should suffice

Reply:
@mohammedanas